config/docker/nginx.conf

```diff
@@ -1,5 +1,5 @@
 server {
-    listen 80 default_server;
+    listen 8080 default_server;
```

Comment:
What's the reason to change the port?

Reply:
In order to run on port 80 (anything below 1024 in fact) you need root privileges. [1] A change to an unprivileged container therefore goes along with changing the port.
Also the new base image uses port 8080 by default. [2]

[1] https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html
[2] https://hub.docker.com/r/nginxinc/nginx-unprivileged

Reply:
Gotcha.

The only problem I see is that this would be a breaking change. On the other hand I'm currently working on releasing shlink-web-client 4.0, so this is a good time to make such change.